MonMaraicher/Features/Map/MapViewModel.swift

```diff
@@ -89,18 +89,20 @@ final class MapViewModel: ObservableObject {
         UIApplication.shared.open(settingsURL)
     }
 
-    private func findNearbyFarmer(from location: CLLocation) -> Farmer? {
+    private func findNearbyFarmer(from location: CLLocation) -> AdressesOperateurs? {
```

Comment:
The naming `findNearbyFarmer` is not good anymore since you are returning addresses, not farmers